src/boot/axios.ts

```diff
@@ -13,17 +13,12 @@ declare module '@vue/runtime-core' {
 // good idea to move this instance creation inside of the
 // "export default () => {}" function below (which runs individually
 // for each client)
-// FIXME: Nested if statements are a code smell, fix it.
 let api: AxiosInstance
-if (process.env.DEV) {
-  if (process.env.MODE === 'capacitor') {
-    api = axios.create({ baseURL: 'http://10.0.2.2:3000' })
-  } else {
-    api = axios.create({ baseURL: 'http://localhost:3000' })
-  }
-} else {
+if(!process.env.DEV)
   api = axios.create({ baseURL: 'https://api.tdl.app' });
-}
+else if(process.env.MODE === 'capacitor')
+  api = axios.create({ baseURL: 'http://10.0.2.2:3000' });
+else api = axios.create({ baseURL: 'https://localhost:3000' });
```

Comment:
I don't know that we really gain much from simply flattening the conditional structure here. That said, it doesn't hurt anything either so I don't mind merging this in as long as we keep the FIXME.